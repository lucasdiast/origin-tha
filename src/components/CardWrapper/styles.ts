import styled from 'styled-components';

export const Container = styled.div`
  background-color: ${(props) => props.theme.colors.white};
  max-width: 560px;
  box-shadow: 0px 16px 32px rgba(30, 42, 50, 0.08);
  border-radius: 8px;
  padding: 40px;
  margin: 0 auto;
`;
