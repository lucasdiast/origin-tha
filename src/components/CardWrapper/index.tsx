import React, { ReactElement } from 'react';

import { Container } from './styles';

interface AuxProps {
  children: React.ReactChild;
}

const CardWrapper = ({ children }: AuxProps): ReactElement => (
  <Container>{children}</Container>
);

export default CardWrapper;
