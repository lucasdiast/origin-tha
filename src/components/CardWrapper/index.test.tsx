import { shallow } from 'enzyme';
import CardWrapper from './index';

describe('CardWrapper', () => {
  it('renders correctly', () => {
    const component = shallow(<CardWrapper>Text</CardWrapper>);

    expect(component).toMatchSnapshot();
  });
});
