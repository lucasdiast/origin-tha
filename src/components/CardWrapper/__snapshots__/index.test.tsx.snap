// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardWrapper renders correctly 1`] = `ShallowWrapper {}`;
